stream_alert/rule_processor/handler.py

```diff
@@ -110,19 +114,22 @@ def _process_alerts(self, classifier, payload, data):
             payload [StreamPayload]: StreamAlert payload object being processed
             data [string]: Pre parsed data string from a raw_event to be parsed
         """
+        is_production = self.env['lambda_alias'] != 'development'
         classifier.classify_record(payload, data)
         if not payload.valid:
-            LOGGER.error('Invalid data: %s\n%s', payload, data)
+            if is_production:
+                LOGGER.error('Log failed to match any defined schemas: %s\n%s', payload, data)
+            self._failed_log_count += 1
             return
 
         alerts = StreamRules.process(payload)
         if not alerts:
             LOGGER.debug('Valid data, no alerts')
```

Comment:
Updating to be: `Processed n valid record that resulted in no alerts` where `n` is the number of records.